drivers/hmis/app/graphql/types/forms/form_definition.rb

```diff
@@ -48,6 +51,54 @@ def definition
       )
     end
 
+    def project_matches
+      # Returns one match per project that this form applies to. (See InstanceProjectMatch for match ranking logic.)
+      # This DOES still return a match for a project if the match is overridden by a more specific rule on another form.
+      # For example, if:
+      # - Intake form A has a rule that specifies that it's used for all projects
+      # - Intake form B has a rule that specifies that it's used for all Emergency Shelter projects
+      # - Intake form C has a rule that specifies that it's used for Project X, an ES project
+      # ...then forms A and B would still return a match for Project X.
+
+      instance_scope = object.instances.active
+
+      # Fetching all projects is slow, so do it only once and pass the scope to project_matches below.
+      project_scope = Hmis::Hud::Project.hmis.preload(:organization)
```

Comment:
the `preload`s should probably be moved to the end once project_scope has been refined